core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -2754,7 +2768,7 @@ class WorkspaceService(protected val ctx: RawlsRequestContext,
 
   def getAccessInstructions(workspaceName: WorkspaceName): Future[Seq[ManagedGroupAccessInstructions]] =
     for {
-      workspaceId <- loadWorkspaceId(workspaceName)
+      workspaceId <- loadV2WorkspaceId(workspaceName)
```

Comment:
Hmmm, another method lacking test coverage.

Reply:
Added in https://github.com/broadinstitute/rawls/pull/2214/commits/215f0fd9af3b924d4f774c464e23262e55d0f283